test/test.ts

```diff
@@ -22,30 +72,73 @@ const getData = async () => {
   try {
     const response = await axios.post(
       "http://localhost:4000/",
-      body,
+      queryBody,
+      axiosConfig
+    );
+    return response;
+  } catch (error) {
+    if (error.response) {
+      console.log(error.response.data);
+      console.log(error.response.status);
+      console.log(error.response.headers);
+    } else if (error.request) {
+      console.log(error.request);
+    } else {
+      console.log(error.message);
+    }
+  }
+};
+
+const getMutation = async () => {
+  try {
+    const response = await axios.post(
+      "http://localhost:4000/",
+      mutationBody,
       axiosConfig
     );
-    console.log("response:");
-    console.log(response);
     return response;
   } catch (error) {
     if (error.response) {
-      console.log("error1");
       console.log(error.response.data);
       console.log(error.response.status);
       console.log(error.response.headers);
     } else if (error.request) {
-      console.log("error2");
       console.log(error.request);
     } else {
-      console.log("Error3", error.message);
+      console.log(error.message);
     }
   }
 };
 
-console.log("hello");
+const test = {
+  id: 0,
+  name: input.name,
+  email: input.email,
+  date_of_birth: new Date(input.date_of_birth),
+  profession: input.profession,
+};
+
+it("query return match the expected", async () => {
+  const queryResponse = await getData();
+  expect(queryResponse.data.data.users).to.be.eq("Hello, Taqos!");
+  console.log("Query users returns the expected!");
+});
 
-it("should return true", async () => {
-  assert.equal(true, true);
-  await getData();
+it("mutation return matches the expected", async () => {
```

Comment:
geralmente nos testes começamos com `it should ...`
```suggestion
it("should create and return user successfully", async () => {
```